programs/drift/src/controller/orders.rs

```diff
@@ -1687,7 +1690,7 @@ pub fn fulfill_perp_order_with_amm(
     fee_structure: &FeeStructure,
     override_base_asset_amount: Option<u64>,
     override_fill_price: Option<u64>,
-    split_with_lps: bool,
+    liquidity_split: AMMLiquiditySplit,
```

Comment:
i feel like `override_base_asset_amount`, `override_fill_price` and `liquidity_split` are all related, maybe we can merge them somehow so it's clearer how they're related

Reply:
think thats a nice future scope